### PR TITLE
Creating UI from design

### PR DESCRIPTION
**Pull request by JavierABarreto**
Create UI views within the application from the design

Search view:
image

Flights view
image

Details view:
image

### CODE REVIEW

File: Frontend/flight-search-amadeus/package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@popperjs/core": "^2.11.8",
     "@reduxjs/toolkit": "^2.2.7",
     "bootstrap": "^5.3.3",
+    "moment": "^2.30.1",
```
Comment (VhugoJc) [inline Frontend/flight-search-amadeus/package.json:17]: Moment is decrypted, please use another library https://momentjs.com/docs/#/-project-status/recommendations/

File: Frontend/flight-search-amadeus/src/components/FlightCards/FlightCard.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+export const FlightCard = () => {
```
Comment (VhugoJc) [inline Frontend/flight-search-amadeus/src/components/FlightCards/FlightCard.tsx:3]: In future: please use object to mock data instead of direct data

Comment (JavierABarreto) [reply Frontend/flight-search-amadeus/src/components/FlightCards/FlightCard.tsx:3]: Will do, just was to put the base idea, I will work on the backend to send the mock data

File: Frontend/flight-search-amadeus/package.json (hunk 1 above)
Comment (JavierABarreto) [reply Frontend/flight-search-amadeus/package.json:17]: Noted

**Pull request merged**